### PR TITLE
Fix subsequent syncs having records to send issue

### LINKED ISSUE #2390: [P2P] Subsequent syncs have records to send
**Describe the bug**
The p2p sync does not seem to finish sending data since subsequent syncs have records to send

**To Reproduce**
1. Open the side menu
2. Click on "Settings"
3. Click on "Transfer Data"
5. Click on "Receive Data" in the new page that opens
6. Initiate data sending on the other device. Repeat from steps 1 to 5. Click on "Send Data"
7. Select the correct receiving device on the sending device
8. When the sending finishes, click on the "Okay" button on the  receiving device
9. Click on the device's back button, and start the process again from step 1 to 7. 

**Expected behavior**

There are still available records to be sent after any p2p sync

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [e.g. Samsung Galaxy A2 emulator]
 - Android version: [e.g. Settings -> About phone -> Android version]
 - Build number: [e.g. Settings -> About phone -> Build number]
 - Version [e.g. 24]

**Additional context**

### PR DESCRIPTION
**Pull request by Rkareko**
populate total count for each type of resource

**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #2390

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [x] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [x] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [x] I have verified any strings visible on UI components are in the `strings.xml` file
- [x] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [x] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [x] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by pld — APPROVED** [review]
lgtm!

File: android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt [hunk 1]
```diff
@@ -82,7 +83,9 @@ constructor(
         } else {
           highestRecordId
         }
-      Timber.e("Sending ${it.resource.resourceType} with id ====== ${it.resource.logicalId}")
+      Timber.e(
```
Comment (ndegwamartin) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt:86]: Any reason we are logging debug information as errors?

File: android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt [hunk 2]
```diff
@@ -82,7 +83,9 @@ constructor(
...
+        "Sending ${it.resource.resourceType} with id ====== ${it.resource.logicalId} and lastUpdated = ${it.resource.meta?.lastUpdated?.time!!}",
+      )
     }
 
     Timber.e("New highest Last updated at value is $highestRecordId")
```
Comment (ndegwamartin) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt:91]: Ditto!

Comment (ndegwamartin) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt:91]: Check for any other occurrences on the file

**Comment by ndegwamartin** [comment]
Also lets bump up the **version code** and **version name** of the `main branch` on this PR so that it matches the release branch, [check out](https://github.com/opensrp/fhircore/blob/main/android/quest/build.gradle.kts#L59-L60)

File: android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt (hunk 1 above)
Comment (Rkareko) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt]: The log level has been updated to `info`

File: android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt (hunk 2 above)
Comment (Rkareko) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt:91]: The log level has been updated to `info`

**Comment by Rkareko** [comment]
> Also lets bump up the **version code** and **version name** of the `main branch` on this PR so that it matches the release branch, [check out](https://github.com/opensrp/fhircore/blob/main/android/quest/build.gradle.kts#L59-L60)

The **version code** and **version name** match with the relase branch as seen [here](https://github.com/opensrp/fhircore/blob/2390-subsequent-p2p-syncs-have-records-to-send-fix/android/quest/build.gradle.kts#L59-L60).

**Review by ndegwamartin — APPROVED** [review]

**Comment by ndegwamartin** [comment]
> > Also lets bump up the **version code** and **version name** of the `main branch` on this PR so that it matches the release branch, [check out](https://github.com/opensrp/fhircore/blob/main/android/quest/build.gradle.kts#L59-L60)
> 
> The **version code** and **version name** match with the relase branch as seen [here](https://github.com/opensrp/fhircore/blob/2390-subsequent-p2p-syncs-have-records-to-send-fix/android/quest/build.gradle.kts#L59-L60).

Sorry, linked the wrong reference - meant [the release branch here](https://github.com/opensrp/fhircore/blob/opensrp-0.2.5/android/quest/build.gradle.kts#L56-L57)

We can always merge as is though and update this later.

**Pull request merged**